tools/docker-compose/supervisor.conf

```diff
@@ -8,24 +8,47 @@ command = make dispatcher
 autorestart = true
 stopasgroup=true
 killasgroup=true
-stdout_logfile=/dev/stdout
-stdout_logfile_maxbytes=0
-stderr_logfile=/dev/stderr
-stderr_logfile_maxbytes=0
+stdout_events_enabled = true
```

Comment:
why was this changed?

Reply:
there seem to be a mix of 

```
stdout_logfile=/dev/stdout
stdout_logfile_maxbytes=0
stderr_logfile=/dev/stderr
stderr_logfile_maxbytes=0
```

and 

```
stdout_events_enabled = true
stderr_events_enabled = true
```

Reply:
this was changed because we are not triggering capture mode with `stdout_capture_maxbytes` as we have it set to 0. Therefore, the change to `stdout_events_enabled` was made as we have no reason to care about the size of the logfile.